Makefile

```diff
@@ -336,7 +336,9 @@ $(SRCBINDIR)/podman$(BINSFX): $(SRCBINDIR) .gopathok $(SOURCES) go.mod go.sum
 		-o $@ ./cmd/podman
 
 $(SRCBINDIR)/podman-remote-static: $(SRCBINDIR) .gopathok $(SOURCES) go.mod go.sum
-	$(GOCMD) build \
+	CGO_ENABLED=0 \
+	CC=$(BINDIR)/musl-gcc \
```

Comment:
Is this needed? I don't think libc is used when cgo is disabled.

Reply:
let me verify without  it.

Reply:
done thanks!